package.json

```diff
@@ -1,50 +1,65 @@
 {
   "name": "PYUSD",
-  "version": "0.0.1",
+  "version": "2.0.0",
   "description": "PayPal-issued USD-collateralized ERC20 stablecoin",
-  "main": "truffle.js",
   "directories": {
     "test": "test"
   },
   "scripts": {
```

Comment:
just to confirm, all the commands below work from a fresh repo clone after npm install?